src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2231,9 +2234,8 @@
         "fixes": {
           "description": "An array of 'fix' objects, each of which represents a proposed fix to the problem indicated by the result.",
           "type": "array",
-          "minItems": 0,
+          "minItems": 1,
```

Comment:
this is wrong, it should be minItems 0 with a default of empty array.

Reply:
ok, @larrygolding@comcast.net please confirm.

3.26.30 fixes property
A result object MAY contain a property named fixes whose value is an array of one or more unique (§3.7.3) fix objects (§3.53).


---
In reply to: [276422386](https://github.com/Microsoft/sarif-sdk/pull/1428#discussion_r276422386) [](ancestors = 276422386)